AnkiDroid/src/main/java/com/ichi2/themes/StyledProgressDialog.java

```diff
@@ -60,6 +60,10 @@ public static MaterialDialog show(Context context, CharSequence title, CharSeque
 
     public static MaterialDialog show(Context context, CharSequence title, CharSequence message,
             boolean cancelable, DialogInterface.OnCancelListener cancelListener) {
+        if ("".equals(title)) {
+            title = null;
+            Timber.w("Invalid title was provided. Using null");
```

Comment:
A `Timber.w` is a warning, in our convention that means "something is definitely wrong, but we may recover and continue".

So, if "something is definitely wrong", how can the same commit be *adding* uses of the style that triggers a warning?

It seems to me that there is nothing wrong with using null and we don't even need an informative message, we are *expecting* to send null and handling it now. At most this should be `Timber.d` now, just so developers aren't surprised but that's it I think
```suggestion
            Timber.d("Invalid title was provided. Using null");
```

Reply:
done

Reply:
The commit is removing `""`, the source of the warning, not adding it.

But `.d` works - we'll pick it up in logcat on a debug machine regardless

Reply:
Ah, sorry was thinking opposite @david-allison-1 - I still don't think it warrants a .w though, .d will note it for devs as mentioned